### PR TITLE
Issue liquid title

### PR DESCRIPTION
**Pull request by skliask**
### Summary

When an `Issue` has liquid content in the title, it is not being displayed properly in some parts of the app.

### Solution

Parse liquid content where the `Issue` titles are being displayed

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -9,6 +9,7 @@
   - Bugs fixes:
     - Evidence: Prevent loading old Evidence template content at the Issue level
     - Methodologies: validate presence of content
+    - Issues: Displays liquid content within the title throughout the app
```
Comment (MattBudz) [inline CHANGELOG:12]:
```suggestion
    - Issues: Render liquid content within the title throughout the app
```

also this should be in :alpha order

File: app/controllers/issues/evidence_controller.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class Issues::EvidenceController < AuthenticatedController
   include ActivityTracking
+  include LiquidEnabledResource
```
Comment (MattBudz) [inline app/controllers/issues/evidence_controller.rb:3]: :alpha

File: app/controllers/projects_controller.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class ProjectsController < AuthenticatedController
   include NotificationsReader
+  include LiquidEnabledResource
```
Comment (MattBudz) [inline app/controllers/projects_controller.rb:3]: :alpha

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -27,6 +27,10 @@ def markup(text, options={})
     result[:output].to_s.html_safe
   end
 
+  def liquid_text(text)
```
Comment (MattBudz) [inline app/helpers/application_helper.rb:30]: is this the best method name for what we are doing here? Does it appropriately describe what this does?

Comment (skliask) [reply app/helpers/application_helper.rb:30]: Maybe `deliquify_text` would be better?

**Pull request closed without merging**